EIPS/eip-1185.md

```diff
@@ -0,0 +1,107 @@
+---
+eip: 1185
+title: Storage of DNS Records in ENS
+author: Jim McDonald <Jim@mcdee.net>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2018-06-26
+requires: 137
+---
+  
+## Abstract
+This EIP defines a resolver profile for ENS that provides features for storage and lookup of DNS records. This allows ENS to be used as a store of authoritative DNS information.
+
+## Motivation
+ENS is a perfect store for DNS records.  It provides the distributed authority of DNS without conflating ownership and serving of information.  With ENS, the owner of a domain has full control over their own DNS records.  Separately, ENS has the ability (through smart contracts) for a domain's subdomains to be irrevocably assigned to another entity.
+
+## Specification
```

Comment:
You should specify the interface IDs for each of the get methods, as well.

Reply:
Added; thanks.